src/backend/joanie/payment/enums.py

```diff
@@ -29,3 +29,7 @@
     # - REFUNDED : invoice balances are equal to zero
     (INVOICE_STATE_REFUNDED, _("Refunded")),
 )
+
+# For invoice name choice (to generate document)
+INVOICE = "invoice"
+INVOICE_NAME_CHOICES = ((INVOICE, _("invoice")),)  # default
```

Comment:
This looks useless ?

Reply:
You are right, I can find the same value on top of that same file.